pages/en/lb4/Creating-components.md

```diff
@@ -228,6 +230,33 @@ export class AuthenticationProvider {
 }
 ```
 
+## Extends Application
+
+When bind a component to an app, you may want to mount the component's propertis 
```

Comment:
- `When bind a component to an app` --> When *binding* a component to an app
- typo: propertis  --> properties